packages/wrangler/src/vectorize/delete.ts

```diff
@@ -0,0 +1,45 @@
+import { readConfig } from "../config";
+import { confirm } from "../dialogs";
+import { logger } from "../logger";
+import { deleteIndex } from "./client";
+import { vectorizeBetaWarning } from "./common";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../yargs-types";
+
+export function options(yargs: CommonYargsArgv) {
+	return yargs
+		.positional("name", {
+			type: "string",
+			demandOption: true,
+			description: "The name of the Vectorize index",
+		})
+		.option("skip-confirmation", {
+			describe: "Skip confirmation",
+			type: "boolean",
+			alias: "y",
+			default: false,
+		})
```

Comment:
We call this type of thing `--force` in a couple other places, so it should probably be the same here for consistency